About-page/index.html

```diff
@@ -0,0 +1,192 @@
+        The event has various speakers from the guest speaker Jurgen Klopp and the show hosted by the Legendary Arsenal Midfielder Patrick Viera and also in attendance the UEFA Presidernt Mitchel Platini.
+      There are lots of opportunities for attendees to share their thoughts and ask questions after the preentation by the various speakers, there are also recreational activities like games etc.
+      </h2>
+    </div>
+    
+
+    <div class="contact-inquiry">
+      <h3>
+        Please contact us via Email for any further questions about FTF22,
+        <br>Hosted by Arsenal Football Club</br>
+        <a href="https://www.Arsenal.com" target="_blank">www.Arsenal.com</a>
+      </h3>
+    </div>
+
+      <div class="about-logo">
+       <h2 class="mobile">
+           Football Tactics Global series <br> 2nd Edition Logo </br>
+        </h2>
+        <h2 class="desktop">
+          Football Tactics Global series 2nd Edition Logo
+       </h2>
+        <div class="line-parent">
+            <hr class="line"/>
+        </div>
+
+        <h3>
+            The Football tactics forum was created through an free online compettition in Qatar 1. Feb to 3.March
+        </h3>
+
+        <div class="logo">
+            <img src="./images/1118.jpg" alt="forum logo">
+        </div> 
+      </div>
+    <section class="past-forum-pictures">
+         <div class="previous-photos">
+            <h2 class="mobile-words">
+                See the past FTF conferences
+            </h2>
+
+            <h3 class="mobile-words2">
+                Take a look at the previous photographs from the FTC Conference that took place in England and Spain.
+            </h3>
+            <h2 class="desktop-words">
+              See the past FTF conferences
+           </h2>
+
+           <h3 class="desktop-words2">
+              Take a look at the previous photographs from the FTC Conference that took place in England and Spain.
+           </h3>
+
+            <div class="photographs photo">
+                
+                  <div class="photograph">
+                    <img src="./images/FOS-22-6.27-Turkish-Super-Lig-BeinSports.jpg" alt="conference-first-photo" class="first-image">
+                
+                  </div>
+
+                 <div class="photograph">
+                    <img src="./images/FSifztXXIAACC_2.jpg" alt="conference-second-image" class="second-image">
+                 </div>
+            </div>
+            <div class="photographs-photo-desktop">
+                
+              <div class="photograph-desktop">
+                <img src="./images/FOS-22-6.27-Turkish-Super-Lig-BeinSports.jpg" alt="conference-first-photo" class="desktop-image">
+            
+              </div>
+
+             <div class="photograph-desktop">
+                <img src="./images/FSifztXXIAACC_2.jpg" alt="conference-second-image" class="desktop-image">
+             </div>
+
+            </div>
+          </div>
+    </section>
```

Comment:
- [ ] [Optional] You have well implemented the required changes said by the previous reviewer but if you look at the `Behance` design, there is equal padding from all sides. Kindly implement it
### your design
![his](https://user-images.githubusercontent.com/81727988/176983752-58862e2d-0a03-471b-9ffa-c1e1c81d9023.png)
### Behance design

![figam](https://user-images.githubusercontent.com/81727988/176983761-6cb9e3bb-7598-4444-86cc-61a31cce6a11.png)